policyengine/countries/us/reform_parameters.yaml

```
@@ -10,6 +10,116 @@ reforms:
         variable: income_tax
         tests:
           - increases_net_income: true
+    abolish_emp_payroll_tax:
```

More verbose but I think it'd be clearer to exactly match the `abolish_` switch names to the variable they abolish.
```suggestion
    abolish_employee_payroll_tax:
```

---

I tried to fix this and it didn't work, even adding to `reform_parameters.yaml` as well. So leaving as-is for now.